include/rppdefs.h

```diff
@@ -37,13 +37,30 @@ SOFTWARE.
 #include <CL/cl.h>
```

Comment:
why is this file changed in this PR?

Reply:
@rrawther This was probably due to merge conflicts and the recent master merge